src/dxvk/dxvk_fence.cpp

```diff
@@ -11,6 +11,12 @@ namespace dxvk {
     typeInfo.semaphoreType = VK_SEMAPHORE_TYPE_TIMELINE;
     typeInfo.initialValue = info.initialValue;
     
+    VkExportSemaphoreCreateInfo exportInfo = { VK_STRUCTURE_TYPE_EXPORT_SEMAPHORE_CREATE_INFO };
+    exportInfo.handleTypes = VK_EXTERNAL_SEMAPHORE_HANDLE_TYPE_D3D12_FENCE_BIT;
```

Comment:
Why do we hard-code the handle type here when we pass it in through the create info?

Reply:
Yeah that was an oversight.